Sources/Mapper.swift

```diff
@@ -332,7 +339,7 @@ public struct Mapper {
      */
     @warn_unused_result
     public func from<T>(field: String, transformation: AnyObject? throws -> T) rethrows -> T {
-        return try transformation(self.JSONFromField(field))
+        return try transformation(try? self.JSONFromField(field))
```

Comment:
This would prevent `JSONFromField` from propagating `MissingFieldError`; is this ok?